frontend/src/containers/projects/index.js

```diff
@@ -1,34 +1,18 @@
-const React = require('react');
-const ReactIntl = require('react-intl');
-const ReactRedux = require('react-redux');
-
-const Button = require('@ui/components/button');
-const Column = require('@ui/components/column');
-const NavLink = require('@ui/components/nav-link');
-const EmptyProjects = require('@components/empty/projects');
-const PropTypes = require('@root/prop-types');
-const Row = require('@ui/components/row');
-const selectors = require('@state/selectors');
-
-const {
-  connect
-} = ReactRedux;
-
-const {
-  FormattedMessage
-} = ReactIntl;
-
-const {
-  orgByIdSelector,
-  projectsByOrgIdSelector
-} = selectors;
-
-const Projects = (props) => {
-  const {
-    org = {},
-    projects = []
-  } = props;
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { connect } from 'react-redux';
+import Button from '@ui/components/button';
+import Column from '@ui/components/column';
+import NavLink from '@ui/components/nav-link';
+import { orgByIdSelector, projectsByOrgIdSelector } from '@state/selectors';
```

Comment:
The rule I followed is: if it fits under 80 chars, keep it the same like, otherwise break the object and do it in a new line (outside of other 1 line imports)